src/core/dbus-manager.c

```diff
@@ -2091,6 +2132,7 @@ const sd_bus_vtable bus_manager_vtable[] = {
         SD_BUS_METHOD("UnsetEnvironment", "as", NULL, method_unset_environment, SD_BUS_VTABLE_UNPRIVILEGED),
         SD_BUS_METHOD("UnsetAndSetEnvironment", "asas", NULL, method_unset_and_set_environment, SD_BUS_VTABLE_UNPRIVILEGED),
         SD_BUS_METHOD("ListUnitFiles", NULL, "a(ss)", method_list_unit_files, SD_BUS_VTABLE_UNPRIVILEGED),
+        SD_BUS_METHOD("ListUnitFilesFiltered", "asas", "a(ss)", method_list_unit_files_filtered, SD_BUS_VTABLE_UNPRIVILEGED),
```

Comment:
this should follow the same naming logic, i.e. if you pick ListUnitsByPatterns() above, then name this ListUnitFilesByPatterns() or so...